app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
No. This is not right. The tz is not even stored in the database correctly. Can you please check why ? It first needs to be stored in the db correctly. `changeToTimezone` is not the issue here.